src/components/NewToken.js

```diff
@@ -0,0 +1,25 @@
+import { getToken, words } from '@the-collab-lab/shopping-list-utils';
+import { useNavigate } from 'react-router-dom';
+import { db } from '../lib/firebase';
+import { doc, setDoc } from 'firebase/firestore';
+
+const NewToken = () => {
+  let navigate = useNavigate();
+
+  function token(products) {
```

Comment:
just for consistency... you guys are using both function and arrow function functions.

Reply:
Ya esta arreglado